### ISSUE TITLE
Mentor Registration for July 2nd, 2022

### ISSUE THREAD

**Issue by Xabadu** [nodeschool/sanfrancisco#301]
Heeeelloooooo 👋!  

It's time for our June event!

<img src="https://media2.giphy.com/media/ki0FCgiaqYRBS/giphy-downsized-medium.gif"/>

(July really, stop with the lies Xabadude)

### Where and when?

Microsoft Reactor is hosting us on July 2nd from 1pm - 5pm

The address is: **680 Folsom St #145, San Francisco, CA 94107** [(map)](https://www.google.com/maps/place/Microsoft+Reactor/@37.7844728,-122.4006337,17z/data=!3m2!4b1!5s0x8085807c4ad752cf:0x8cce11dc92be2213!4m5!3m4!1s0x8085807c4ac95771:0x97838fa44b46f66a!8m2!3d37.7844686!4d-122.398445) -- Comment to get involved or ask questions.

### COVID-19 Guidelines

We are working with the Microsoft Reactor folks to have a welcoming and inclusive experience for everyone, and we want you all to feel as safe as possible by following all the recommended guidelines for the venue. That means:

- We will ask for proof of vaccination for all attendees and mentors.
- Microsoft Reactor will require everyone to fill a form before entering confirming you haven't had any COVID-19 symptoms.

If you have any questions around this, please don't hesitate to ask!

### Is this your first time? No worries, we got you!

If this is your first time mentoring and you're not sure you're ready, we're sure you are! Just leave us a comment and we'll help you in any way we can.

Mentors should:
- Review the [Mentor Best Practices](https://github.com/nodeschool/organizers/wiki/Event-Mentor-Best-Practices) wiki page.
- Arrive 30 minutes early (12:30p) so we can get set up and have our mentor meeting.

Mentors can also optionally:

- Have already completed the [core NodeSchool workshoppers](http://nodeschool.io/#workshopper-list) before the event.
- Having also completed elective workshoppers is a great bonus.

We're in the process of renovating our learning resources for NodeSchool, so these last two are no longer required for mentors. As long as you feel comfortable with JavaScript/Node.js (and optionally Web Dev in general), you're all set!

### Confirmed mentors:
- @Zahidul-Islam
- @alexpenadev
- @seraficaj
- @creedarky
- @ch264
- @olivier-martin-sf
- @sumeet
- You?
- You?
- You?

### Organizers:
- @tlhunter
- @Xabadu

Thanks for joining! Looking forward to seeing you all! 🙌 

Attendee registration link:  https://www.meetup.com/sfnodeschool/events/286431888

**Comment by Xabadu** [comment]
@ch264 @alexpenadev @Fauntleroy @kevinsqi @tauseefk @shiya @cantremember @creedarky @pangperry @Abimzy @Zahidul-Islam @jorgepvenegas @seraficaj @sorrelbri @wongco @anchnk  ⬡.js? 🍕? 👍?

**Comment by Zahidul-Islam** [comment]
Count me in...

**Comment by alexpenadev** [comment]
Count me in too...

**Comment by seraficaj** [comment]
I'll be there! 😄

**Comment by creedarky** [comment]
I'll be there 
> Count me in too...

![](https://media3.giphy.com/media/dsGcvvBPzXJ9KSnLPg/giphy.gif?cid=ecf05e47e8pntjknb3xqaug73wpobhhzi6iaxtr3qz68svza&rid=giphy.gif&ct=g)

**Comment by Xabadu** [comment]
@creedarky nooooice, you're in! 

<img src="https://media1.giphy.com/media/yoJC2pskiOu9UgcTu0/giphy-downsized-medium.gif"/>

**Comment by ch264** [comment]
Me please :D @[seraficaj](https://github.com/seraficaj) I look forward to seeing you again! It has been ages

**Comment by Xabadu** [comment]
@ch264 hell yeah! Thank you for joining! 🎉 

![roars](https://user-images.githubusercontent.com/3161019/172920108-753b39da-cb7e-486c-a40e-8dfac11a0651.gif)

**Comment by olivier-martin-sf** [comment]
Hey Looks like I will be able to make it happen this time!

**Comment by sumeet** [comment]
can i be added to the mentors list? thank you!

**Comment by Xabadu** [comment]
@sumeet you got it! 

<img src="https://media0.giphy.com/media/3o7TKGy6TBUPrjtQLC/giphy.gif"/>

**Comment by tauseefk** [comment]
somehow I didn't see this until now.. Totally down to mentor some mentees!

![https://media.giphy.com/media/3oKIP9hfEsvPHxfIk0/giphy-downsized.gif](https://media.giphy.com/media/3oKIP9hfEsvPHxfIk0/giphy-downsized.gif)

**Comment by Xabadu** [comment]
Hey folks! 👋 

Bit of sad news 💔 . We've decided to cancel this event and meet again on the 30th for (the real) July event (no worries, we'll make announcements soon!).

There's a few different things going on for which we had to make this decision, but to be super transparent, one of the main reasons is low registration numbers (considering that on top of that, normally around half of the signups actually show up), most likely due to the long weekend. Even though the folks at the Reactor are fantastic and super accommodating, we prefer not to make them have to staff and work over that long weekend if numbers are gonna be that low and instead reconvene for the next one.

Apologies for the sadness of not seeing each other until then, thank you for your understanding.

Stay safe! ✌️ 

p.s: Closing this for now, but I'll still monitor this issue, so if you have any questions or comments, feel free to drop them here, or on our [discord](https://discord.nodeschoolsf.dev)
